packages/message-encryption/src/symmetric.ts

```diff
@@ -79,6 +80,8 @@ class Encoder implements IEncoder {
 }
 
 export interface EncoderOptions extends BaseEncoderOptions {
+  /** Indicates if autosharding is enabled */
+  autosharding?: boolean;
```

Comment:
not sure i understand the relevance of having this flag on `EncoderOptions`

Reply:
these shouldn't be here; removed from here and `ecies.ts` in latest commit